doc/source/ray-contribute/getting-involved.rst

```diff
@@ -105,13 +105,13 @@ The full suite of tests is too large to run on a single machine. However, you ca
 
 .. code-block:: shell
 
-    pytest -v -s python/ray/tests/test_basic.py
+    python -m pytest -v -s python/ray/tests/test_basic.py
```

Comment:
@clarkzinzow review?

Reply:
Once we're on pytest >= 7.0.0, we should remove this and opt for the `pythonpath` configuration value: https://docs.pytest.org/en/7.1.x/reference/reference.html#confval-pythonpath

Do we need this if we're pinning pytest in 7.0.1 in this PR?

https://github.com/ray-project/ray/pull/26334